tests/e2e/utils/config.go

```diff
@@ -75,7 +75,7 @@ func RandomInt(min, max int) int {
 	return min + rand.Intn(max-min)
 }
 
-//function to Generate Random string
+//GetRandomString to Generate Random string
```

Comment:
we should be careful in building efficient random string generator functions. Look at this blog for more ideas : https://medium.com/@kpbird/golang-generate-fixed-size-random-string-dd6dbd5e63c0.
is it required to introduce the time while generating the random number ? Please check how does it perform with some benchmarking. functions like these could possibly slow down our tests if we spend more time here.
let's optimize this code.

Reply:
@rohitsardesai83 ,As sugested, Gone  through the link, used light weight method for generating the random string.

Reply:
@rohitsardesai83 , i have gone through the blog, tried with 2 different types of random string generate functions which are 2 quickest as per the blogs benchmarking, but when i tested, randomly the functions are giving previously generated strings, i feel those are not so consistent to use. so i reverted the change again.
i will once again go through this blog, others to use more simplest or quickest function to random genrate strings for our tests. 
but for now, im keeping with the same function. will be fixing in subsequent PR.